scrapy/exporters.py

```diff
@@ -96,9 +100,13 @@ class JsonItemExporter(BaseItemExporter):
     def __init__(self, file, **kwargs):
         self._configure(kwargs, dont_fail=True)
         self.file = file
-        self.encoder = ScrapyJSONEncoder(**kwargs)
+        self.encoder = ScrapyJSONEncoder(ensure_ascii=self.ensure_ascii, **kwargs)
         self.first_item = True
 
+    def _configure(self, options, dont_fail=False):
+        self.ensure_ascii = options.pop('ensure_ascii', True)
```

Comment:
Yup, I got that now. I'm using a custom feedexporter, so I could pass ensure_ascii option myself.
I'll fix it later today =)